sjsonnet/src-native/sjsonnet/Platform.scala

```diff
@@ -0,0 +1,15 @@
+package sjsonnet
+
+object Platform {
+  def gzipBytes(b: Array[Byte]): String = GZIP.gzipBytes(b)
+  def gzipString(s: String): String = {
+    gzipBytes(s.getBytes())
+  }
+  def xzBytes(b: Array[Byte]): String = throw new Exception("XZ not implemented in Scala Native")
+  def xzString(s: String): String = throw new Exception("XZ not implemented in Scala Native")
```

Comment:
XZ is actually implementable in Scala Native using lzma. I will do it in a follow up PR